go/felix/iptables/table_test.go

```diff
@@ -357,6 +374,13 @@ var _ = Describe("Table with a dirty dataplane", func() {
 				// This rule will get rewritten because its hash is incorrect.
 				"-m comment --comment \"cali:1234567890ksamdl\" --jump DROP",
 			},
+			"non-calico-insert": {
+				// This rule will get cleaned up because we don't insert any rules
+				// into this chain.
+				"--jump ACCEPT",
+				"-m comment --comment \"cali:hecdSCslEjdBPfds\" --jump DROP",
```

Comment:
How is this a non-Calico insert if it has a Calico hash comment?  (Probably just needs a tad more commenting.)